st2client/tests/base.py

```diff
@@ -86,6 +86,8 @@ def setUp(self):
             if var in os.environ:
                 del os.environ[var]
 
+        os.environ['ST2_SKIP_CLI_CONFIG'] = '1'
```

Comment:
I'd leave the env variable thing undocumented for now. I am pretty sure people will abuse it by using `export` and will complain on IRC. I do like --skip-config.